src/components/Navigation/NavBar.scss

```diff
@@ -122,9 +122,6 @@ $widget-height: 50px;
 }
 
 .spacing {
-  width: 100%;
-  height: $logo-height - $widget-height;
-
```

Comment:
Skjønte ikke vitsen med disse to linjene? Hadde kanskje ikke trengt å fjerne de, men merket ingen forskjell